src/content/docs/dropins/order/tutorials/order-cancellation.mdx

```diff
@@ -0,0 +1,321 @@
+---
+title: Order Cancellation
+description: Learn the Order Cancellation workflow.
+---
+
+import Aside from '@components/Aside.astro';
+import Diagram from '@components/Diagram.astro';
+import OptionsTable from '@components/OptionsTable.astro';
+import { Steps } from '@astrojs/starlight/components';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+
+The Order dropin enables both logged in and guest users to cancel an order.
+
+## Big picture
+
+The order cancellation workflow is as follows:
+
+<Steps>
+
+1. The shopper selects an order to cancel. Guest users must use the order search form to locate the order. Logged-in customers can select an order from their order history or use the search form.
+
+1. The shopper submits the cancellation form after selecting a cancellation reason.
+
+1. If the shopper is a logged-in customer, the order is `Canceled` immediately. Otherwise, the order remains in status `Pending` until the guest clinks a link in an confirmation email.
```

Comment:
```suggestion
1. If the shopper is a logged-in customer, the order is `Canceled` immediately. Otherwise, the order status remains `Pending` until the guest clicks a link in a confirmation email.
```